### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by bolinocroustibat**
- Add architecture schema to docs
- Small README fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ Since it's called _hydra_, it also has mythical powers embedded:
 - if the remote resource is a CSV, convert it to a PostgreSQL table, ready for APIfication
 - send crawl and analysis info to a udata instance
 
+## Architecture schema
+
+The architecture for the ful workflow is the following:
```
Comment (maudetes) [inline README.md:14]:
```suggestion
The architecture for the full workflow is the following:
```

**Review by maudetes — APPROVED** [review]
:pray:

**Comment by bolinocroustibat** [comment]
@maudetes Could you change my GitHub permissions so I can merge and push on the repo? Thanks!

**Comment by maudetes** [comment]
> @maudetes Could you change my GitHub permissions so I can merge and push on the repo? Thanks!

Should be ok, let me know if it does not work

**Pull request merged**